apps/reaper/lib/reaper/event/handlers/ingestion_update.ex

```diff
@@ -0,0 +1,105 @@
+defmodule Reaper.Event.Handlers.IngestionUpdate do
+  @moduledoc false
+  require Logger
+
+  import SmartCity.Event, only: [data_extract_start: 0, file_ingest_start: 0]
+
+  alias Quantum.Job
+  alias Reaper.Collections.Extractions
+
+  @instance_name Reaper.instance_name()
+
+  @cron_conversions %{
+    86_400_000 => "0 6 * * *",
+    3_600_000 => "0 * * * *",
+    30_000 => "*/30 * * * * * *",
+    10_000 => "*/10 * * * * * *"
+  }
```

Comment:
This, and everything that uses it, can be deleted. We no longer support integer cadences, and haven't for ~3 years now. Either make a new issue to remove it, or take it out as part of this PR

Reply:
Sounds like a plan! We will delete as part of this PR!